tests.py

```diff
@@ -46,20 +47,31 @@ def test_flake8(self):
         assert exit_status == 0
 
     def test_additional_management_command_options(self):
-        exit_status, output = run_silently(
-            'python manage.py behave --help')
+        exit_status, output = run_silently('python manage.py behave --help')
         assert exit_status == 0
         assert (LF + '  --use-existing-database' + LF) in output
 
-    def test_command_should_exit_zero_if_passing(self):
-        exit_status, output = run_silently(
-            'python manage.py behave --tags ~@failing')
-        assert exit_status == 0
+    @patch('behave_django.management.commands.behave.behave_main', return_value=0)  # noqa
+    @patch('sys.exit')
+    def test_command_should_exit_zero_if_passing(self,
+                                                 mock_sys_exit,
+                                                 mock_behave_main):
+        # If the exit status returned by behave_main is 0, make sure sys.exit
+        # does not get called
+        run_management_command('behave', dry_run=True)
```

Comment:
Right, now I understand. You're running the behave tests in the Travis file anyway.

If you split up the `script` line into 2 executions (`python manage.py behave`, `python tests.py`) in [.travis.yml](https://github.com/mixxorz/behave-django/blob/master/.travis.yml) instead of concatenating them with `&&` then it's easier to examine the output of the separate commands on Travis-CI. May be useful in case of errors.